### ISSUE TITLE
Togglebuttons: <li> elements must be contained in a <ul> or <ol> [accessibility]

### ISSUE THREAD

**Issue by Aellynd** [formkit/formkit#1334]
### Reproduction

https://formkit.link/cd719fb7236742ebf0d39717ad863c9e

### Describe the bug

Hi, we just upgraded to @formkit/pro@0.122.19 and noticed a new accessibility issue was introduced in this finding as a result of a fix in #1326:
- \<li> elements must be contained in a \<ul> or \<ol> (User Impact: Serious, Guidelines: WCAG 2.1 (A), WCAG 2.0 (A), WCAG 2.2 (A), EN 301 549)

This is due to the \<ul> having role="group".

I hope it is clear, otherwise feel free to ask me more questions.
Thanks in advance!

### Environment

• Windows 10 Pro
• Chrome 124.0.6367.61

**Comment by sashamilenkovic** [comment]
@Aellynd This should be addressed at FormKit Pro v0.122.20. I validated that this passes checks on lighthouse. Please let us know if you see any more issues. Thanks.

**Comment by Aellynd** [comment]
Thank you @sashamilenkovic! Again, the quick action and response is much appreciated. I can confirm there are no more accessibility issues for the toggle buttons 🥇
